.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
+with multi-package repos, or single-package repos to help you version and publish your code. You can
+find the full documentation for it [in our repository](https://github.com/changesets/changesets)
+
+We have a quick list of common questions to get you started engaging with this project in
+[our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
```

Comment:
Nice addition! so this is used through `yarn` to build the packages for JS, TS and React. Right?

Reply:
This one is automatically generated by using command `yarn changeset`. Leave it as it is so that changeset can find this folder with its auto-generated README.md inside `.changeset` folder.

Reply:
nice!